File: Aula20/revolver.py
class Revolver:
    qtmunicao = 0
    qtMaxima = 0

    def Atirar(self):
        if self.qtMunicao > 0:
            self.qtMunicao -= 1

    def Atirar(self):
        if self.qtMunicao < self.qtMaxima:
            self.qtMunicao += 1

    def mostrar (self):
        print(self.qtMunicao)

    

File: Aula2/com/aula/Revolver.java
package com.aula;

public class Revolver {
    int qtMunicao , qtMaxima;


    Revolver(int qtMaxima, int qtMunicao){
        this.qtMaxima = qtMaxima;
        this.qtMunicao = qtMunicao;

    }
    void atirar(){
        if(qtMunicao > 0){
            qtMunicao --;
            System.out.println("POW");
        }
    }
     void recarregar(){
        if(qtMunicao < qtMaxima){
            qtMunicao ++;
        }
    }
    void mostrar(){
        System.out.println(qtMunicao);
    }
    
}
